mojo-data/mojos/rulefit/cars/v1.00/rulefit_v1.00_cars_multiparams.txt

```diff
@@ -0,0 +1,46 @@
+2
+2
+2
+2
+2
+2
+java.lang.ArrayIndexOutOfBoundsException
+java.lang.ArrayIndexOutOfBoundsException
+-1
+null
+-1
+null
+-1
+null
+-1
+null
+-1
+null
+-1
+null
+-1
+null
+java.lang.ArrayIndexOutOfBoundsException
+java.lang.ArrayIndexOutOfBoundsException
+java.lang.ArrayIndexOutOfBoundsException
+java.lang.ArrayIndexOutOfBoundsException
+null
```

Comment:
caused by 

mojoModel.getNumClasses(7) == -1); // java.lang.ArrayIndexOutOfBoundsException: 7
mojoModel.getDomainValues(7) == null); // java.lang.ArrayIndexOutOfBoundsException: 7
mojoModel.getNumClasses(8) == -1); //java.lang.ArrayIndexOutOfBoundsException: 8
mojoModel.getDomainValues(8) == null); //java.lang.ArrayIndexOutOfBoundsException: 8

so this is ok because there is only 7 features

Reply:
the same in other outputs ending by _multiparams.txt